contrib/sly-package-fu.el

```diff
@@ -317,4 +332,126 @@ symbol in the Lisp image if possible."
 
 (defalias 'sly-export-structure 'sly-export-class)
 
+;; 
+;; Dealing with import-from
+;;
+;; MELPA version: (buffer-substring-no-properties point1 pount2)
+;; ~/.emacs.d/elpa/sly-20171111.1604/contrib/sly-package-fu.el
+
+(defun sly-search-import-from (package)
+  ;; Suppose, we are in the defpackage sexp
```

Comment:
Please rename this `sly-package-fu--search-import-from`. Also, please clarify the comment. Do you mean "Assumes we are inside the defpackage sexp"? If so where exactly is point assumed to be?

Reply:
Done.